plugins/woocommerce/client/legacy/css/_variables.scss

```diff
@@ -14,8 +14,8 @@ $primarytext:       desaturate(lighten($primary, 50%), 18%) !default;    // Text
 $secondary:         desaturate(lighten($primary, 40%), 21%) !default;    // Secondary buttons
 $secondarytext:     desaturate(darken($secondary, 60%), 21%) !default;   // Text on secondary color bg
 
-$highlight:         adjust-hue($primary, 150deg) !default;               // Prices, In stock labels, sales flash
-$highlightext:      desaturate(lighten($highlight, 50%), 18%) !default;  // Text on highlight color bg
+$highlight:         adjust-hue(darken($primary, 18%), 150deg) !default;  // Prices, In stock labels, sales flash
+$highlightext:      desaturate(lighten($highlight, 80%), 18%) !default;  // Text on highlight color bg
```

Comment:
Adjustment for Twenty Twenty-Four theme.
**Before**
![image](https://github.com/user-attachments/assets/83e182eb-f42d-4992-abd5-aaa2d9ae5a56)

**After**
![Screenshot 2024-10-16 at 18 19 03](https://github.com/user-attachments/assets/2f1375ba-3ba6-4654-8995-b63418c1bd25)